.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+inherit_gem:
+  rubocop-config-umbrellio: lib/rubocop.yml
+
+AllCops:
+  TargetRubyVersion: 2.6.5
```

Comment:
we should use the minimal supported ruby version here (2.4.7 at this moment)

Reply:
Done.